Sources/Public/CalendarView.swift

```diff
@@ -1032,9 +1107,7 @@ extension CalendarView {
     focusedAccessibilityElement = element
 
     if let contentView = element as? UIView, let itemView = contentView.superview as? ItemView {
-      itemTypeOfFocusedAccessibilityElement = visibleViewsForVisibleItems
-        .first { _, visibleView in itemView === visibleView }?
-        .key.itemType
```

Comment:
https://github.com/airbnb/HorizonCalendar/pull/210/files#diff-3fd1421d744edc270d08ce53e359091bddf70fbbafd5ab5f3cde98e8f1032cd1R61

no need to do an O(n) search here anymore : )